datadog-monitors-monitor-handler/src/main/java/com/datadog/monitors/monitor/ReadHandler.java

```diff
@@ -54,7 +54,10 @@
                 .build();
         }
 
+        final String monitorURL = model.getDatadogCredentials().getApiURL() + "/api/v1/monitor/" + model.getId().toString();
```

Comment:
Any risks to get NPEs along the way here?

Reply:
Also regarding the returned type since we could have both an API Url, an Http URL but also we may want the monitor ID only. Would it make sense to return an object with everything in it instead of a String?